src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  &__container {
+    display: grid;
+    grid-template-columns: repeat(4, #{$card-width});
+    justify-content: center;
+    gap: 48px;
+    padding: 50px 0;
+
+    @media screen and (max-width: 1023px) {
+      grid-template-columns: repeat(3, #{$card-width});
+    }
+
+    @media screen and (max-width: 767px) {
+      grid-template-columns: repeat(2, #{$card-width});
+    }
+
+    @media screen and (max-width: 487px) {
+      grid-template-columns: #{$card-width};
+    }
```

Comment:
You could use CSS variables for card width, and in the media query you could change its value

Reply:
fixed